source/pigeonhole-man/sieve-dump.1.rst.in

```diff
@@ -0,0 +1,105 @@
+.. _man-1_sieve-dump:
+
+==========
+sieve-dump
+==========
+
+NAME
+====
+
+sieve-dump - Pigeonhole's Sieve script binary dump tool
+
+SYNOPSIS
+========
+
+**sieve-dump** [*options*] *sieve-binary* [*out-file*]
+
+DESCRIPTION
+===========
+
+The **sieve-dump** command is part of the Pigeonhole Project (:man:`pigeonhole(7)`),
+which adds Sieve (`RFC 5228 <http://tools.ietf.org/html/rfc5228/>`_) and
```

Comment:
IETF tool URLs are deprecated and now redirect to rfc-editor. Should probably update these.

Reply:
see #614